src/app/[locale]/projects/view/page.tsx

```diff
@@ -0,0 +1,97 @@
+
+"use client"
+
+import { Col, Row, ListGroup, Tab, Button } from 'react-bootstrap'
+import { SummaryView, AdministrationView, LinkProjectsModal, 
+            LicenseClearingView, ObligationsView, ECCView,
+            VulnerabilityStatusView, AttachmentsView, VulnerabilitiesView,
+            ChangelogView } from "@/components/sw360"
+import { useState } from 'react'
+
+export default function ViewProjects() {
+
+    const [showLinkProjectsModal, setShowLinkProjectsModal] = useState(false)
+
+    return (
+        <>
+            <LinkProjectsModal show={showLinkProjectsModal} setShow={setShowLinkProjectsModal} />
+            <div className="ms-5 mt-2">
+                <Tab.Container defaultActiveKey="summary">
+                    <Row>
+                        <Col sm="auto" className="me-3">
+                            <ListGroup>
+                                <ListGroup.Item action eventKey="summary">
+                                    <div className="my-2">Summary</div>
+                                </ListGroup.Item>
+                                <ListGroup.Item action eventKey="administration">
+                                    <div className="my-2">Administration</div>
+                                </ListGroup.Item>
+                                <ListGroup.Item action eventKey="licenseClearing">
+                                    <div className="my-2">License Clearing</div>
+                                </ListGroup.Item>
+                                <ListGroup.Item action eventKey="obligations">
+                                    <div className="my-2">Obligations</div>
+                                </ListGroup.Item>
+                                <ListGroup.Item action eventKey="ecc">
+                                    <div className="my-2">ECC</div>
+                                </ListGroup.Item>
+                                <ListGroup.Item action eventKey="vulnerabilityTrackingStatus">
+                                    <div className="my-2">Vulnerability Tracking Status</div>
+                                </ListGroup.Item>
+                                <ListGroup.Item action eventKey="attachments">
+                                    <div className="my-2">Attachments</div>
+                                </ListGroup.Item>
+                                <ListGroup.Item action eventKey="attachmentUsages">
+                                    <div className="my-2">Attachment Usages</div>
+                                </ListGroup.Item>
+                                <ListGroup.Item action eventKey="vulnerabilities">
+                                    <div className="my-2">Vulnerabilities</div>
+                                </ListGroup.Item>
+                                <ListGroup.Item action eventKey="changeLog">
+                                    <div className="my-2">Change Log</div>
+                                </ListGroup.Item>
```

Comment:
Add strings as translatable way.

Add in the header this two imports:
```typescript
import { useTranslations } from 'next-intl'
import { COMMON_NAMESPACE } from '@/object-types/Constants'
```

Add this inside page function:
```typescript
const t = useTranslations(COMMON_NAMESPACE)
```

And then do strings like that:
```typescript
t("String"
<div classname="my-2">{t("Attachments")}</div>
```